docs/getting-started/guides/libs/crypto/ec256.md

```diff
@@ -0,0 +1,550 @@
+      <td>uint256</td>
+      <td>The scalar u</td>
+    </tr>
+  </tbody>
+</table>
+
+### jMultShamir2
+
+```solidity
+function jMultShamir2(
+    EC256.Curve memory ec,
+    EC256.JPoint memory jPoint1_,
+    EC256.JPoint memory jPoint2_,
+    uint256 scalar1_,
+    uint256 scalar2_
+) internal pure returns (EC256.JPoint memory jPoint3_);
+```
+
+#### Description
+
+Simultaneous double-scalar multiplication: R = u1*P1 + u2*P2 via Strauss–Shamir.
+
+Returns the Jacobian representation of result point R.
+
+##### Parameters:
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>ec</code></td>
+      <td>struct EC256.Curve</td>
+      <td>The curve parameters</td>
+    </tr>
+    <tr>
+      <td><code>jPoint1</code></td>
+      <td>struct EC256.JPoint</td>
+      <td>The first Jacobian point P1</td>
+    </tr>
+    <tr>
+      <td><code>jPoint2</code></td>
+      <td>struct EC256.JPoint</td>
+      <td>The second Jacobian point P2</td>
+    </tr>
+    <tr>
+      <td><code>scalar1</code></td>
+      <td>uint256</td>
+      <td>The first scalar u1</td>
+    </tr>
+    <tr>
+      <td><code>scalar2</code></td>
+      <td>uint256</td>
+      <td>The second scalar u2</td>
+    </tr>
+  </tbody>
+</table>
+
+### jAddPoint
+
+```solidity
+function jAddPoint(
+    EC256.Curve memory ec,
+    EC256.JPoint memory jPoint1_,
+    EC256.JPoint memory jPoint2_
+) internal pure returns (EC256.JPoint memory jPoint3_);
+```
+
+#### Description
+
+Adds two Jacobian points: R = P1 + P2.
+
+Returns the Jacobian representation of result point R. 
+
+##### Parameters:
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+     <tr>
+      <td><code>ec</code></td>
+      <td>struct EC256.Curve</td>
+      <td>The curve parameters</td>
+    </tr>
+    <tr>
+      <td><code>jPoint1</code></td>
+      <td>struct EC256.JPoint</td>
+      <td>The first Jacobian point P1</td>
+    </tr>
+    <tr>
+      <td><code>jPoint2</code></td>
+      <td>struct EC256.JPoint</td>
+      <td>The second Jacobian point P2</td>
+    </tr>
+  </tbody>
+</table>
+
+### jDoublePoint
+
+```solidity
+function jDoublePoint(
+    EC256.Curve memory ec,
+    EC256.JPoint memory jPoint1_
+) internal pure returns (EC256.JPoint memory jPoint2_);
+```
+
+#### Description
+
+Doubles a Jacobian point: R = 2*P.
+
+Returns the Jacobian representation of result point R. 
+
+##### Parameters:
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>ec</code></td>
+      <td>struct EC256.Curve</td>
+      <td>The curve parameters</td>
+    </tr>
+     <tr>
+      <td><code>jPoint</code></td>
+      <td>struct EC256.JPoint</td>
+      <td>The Jacobian point P to double</td>
+    </tr>
+  </tbody>
+</table>
+
+## Example
```

Comment:
I'd make some more demonstrative example instead like showing `2G = G+G`, where `G` is a basepoint.